Remotion/Web/Development.WebTesting/ScreenshotCreation/BrowserContentLocators/EdgeBrowserContentLocator.cs

```diff
@@ -104,20 +104,16 @@ private AutomationElement ResolveByChangingWindowTitle (IWebDriver driver, IRead
 
     private Rectangle ResolveBoundsFromWindow (AutomationElement window)
     {
-      var element = RetryUntilValueChanges (
-          () => window.FindFirst (
-              TreeScope.Subtree,
-              new AndCondition (
-                  new PropertyCondition (AutomationElement.ControlTypeProperty, ControlType.Document),
-                  new PropertyCondition (AutomationElement.FrameworkIdProperty, c_edgeFrameworkID))),
+      var contentElement = RetryUntilValueChanges (
+          () => GetContentElement (window),
           null,
           5,
           TimeSpan.Zero);
 
-      if (element == null)
+      if (contentElement == null)
         throw new InvalidOperationException ("Could not find the content window of the found Edge browser window.");
 
-      var rawBounds = RetryUntilValueChanges (() => element.Current.BoundingRectangle, Rect.Empty, 3, TimeSpan.FromMilliseconds (100));
+      var rawBounds = RetryUntilValueChanges (() => GetContentElement (window).Current.BoundingRectangle, Rect.Empty, 3, TimeSpan.FromMilliseconds (100));

```

Comment:
Okay, for some unknown reason it is required to get the contentElement, throw if it is null, then discard the contentElement and get it a second time. Can we make the code more explicit or otherwise we will need to add inline comments to prevent future-dev from removing this obvious duplication.